cmd/push_images.go

```diff
@@ -0,0 +1,76 @@
+// TODO: Should be de-duped with similar code in features repo
```

Comment:
FWIW, this is why this repo depends on the features repo - I wanted to dedupe the whole build worker on any-version of any-SDK.

Reply:
Yeah for sure, I think we'll want to eventually figure out what we want the separation to be and where everything goes per our standup discussion at some point, probably after this initial CICD effort is running properly.